### PR TITLE
fixed quota api bug

### PR DESCRIPTION
**Pull request by jiho-kr**
Fixes a bug where the endpoint data broken when using the quota API.

### CODE REVIEW

**Comment by jiho-kr** [comment]
consul key get/set test complete.

consul.kv.get(k).then(res => res && JSON.parse(res.Value));

This Bug was caused by missing ".Value".

**Comment by jiho-kr** [comment]
consul key get returns string type.
add JSON.parse(res.Value)

**Comment by jiho-kr** [comment]
release_candidate

File: src/app.ts [hunk 1]
```diff
@@ -285,8 +285,9 @@ public unregisterIsland(name: string, value: { hostname: string, port: any, patt
       return this.getKey(`/${this.ns}.${ENDPOINT_PREFIX}${endpointName}`).then(res => {
            if (!res)
                throw new Error('not found endpoint');
-           res.quota = opts;
-           return this.setKey(`/${this.ns}.${ENDPOINT_PREFIX}${endpointName}`, res);
+           let value = JSON.parse(res.Value);
```
Comment (heycalmdown) [inline src/app.ts]: `value` can be `const` safely.

**Pull request merged**